submissioncclicenses.md

```diff
@@ -115,11 +115,27 @@ Some licenses contain questions, e.g. standard retrieved from https://api.creati
 The fields are questions to be answered when assigning the license
 
 ## Search CC License 
-**/api/config/submissioncclicenses/search/rightsByQuestions**
+**/api/config/submissioncclicenseurl/search/rightsByQuestions**
```

Comment:
@KevinVdV : I just realized I gave you a bad pathname for this new endpoint. Per REST best practices, the pathname should be plural. So, this endpoint should be:
`/api/config/submissioncclicenseurls` (notice the `s` on the end)

Or, as noted, we could move all these under `/api/submission`, in which case this would be `/api/submission/cclicenseurls`

Reply:
I renamed the endpoint to the plural form, but changing all these endpoints to use `/api/submission` would be a break with how the other submission configuration endpoints work, take a look at:
* https://github.com/DSpace/Rest7Contract/blob/master/submissionforms.md
* https://github.com/DSpace/Rest7Contract/blob/master/submissiondefinitions.md
* https://github.com/DSpace/Rest7Contract/blob/master/submissionsections.md
* ...

Reply:
Right, currently we have inconsistency in where submission features are grouped in the REST contract.  Some are under `/api/config/submission*`, while others are under `/api/submission/` (e.g. https://github.com/DSpace/Rest7Contract/pull/128/ , workspaceitem and workflowitem).  I was pointing out that these CC License endpoints could be in either location.

But, I won't push this. I'm simply beginning to suspect it may be best to move all submission features (including config based) under `/api/submission/`.  But, we can always do that at a later time in a future PR.